packages/components/addon/components/hds/table/index.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+
+/**
+ * @class HdsTable
+ */
+export default class HdsTableIndexComponent extends Component {
+  /**
+   * Sets the data source that gets yielded by the `:body` named block.
+   * @argument model
+   * @type {array}
+   */
```

Comment:
Could we add a `@key` argument that gets plumbed into the inner `{{#each}}`?

Reply:
@jgwhite I think we could, let me try it out!